concordia/static/js/action-app/components.js

```diff
@@ -265,7 +260,6 @@ class AssetListItem {
         this.el.classList.add('asset', 'border');
         this.el.dataset.image = thumbnailUrl;
         this.el.dataset.id = assetData.id;
-        this.el.dataset.difficulty = assetData.difficulty;
```

Comment:
Do we need this to maintain sorting functionality?

Reply:
We used to but 16261847ef38b4bd36bf5fc2d553c0d1e9769706 removed that dependency because we're now using the re:dom keyed-list model which always receives our asset record instead of the DOM element. I removed it on the theory that it was work we weren't currently benefiting from.